stories/VideoHero.stories.js

```diff
@@ -0,0 +1,35 @@
+import VideoHero from "../client/src/components/VideoHero.js";
```

Comment:
Hi Monlka, same as above, just checking is this part of the header and footer PR?

Reply:
you're right. its not. can we still merge it?

Reply:
i am goin to remove it from this branch and push it from the videostory branch